tinyagentos/routes/store_install.py

```
@@ -799,38 +802,40 @@ async def install_app(request: Request):
-                            ),
-                            "install_id": install_id,
-                        },
-                        status_code=403,
-                    )
+
+        def _toctou_reverify():
+            disk_path = _manifest_dir / "manifest.yaml"
+            try:
+                import yaml as _yaml
+                on_disk = _yaml.safe_load(disk_path.read_text()) if disk_path.exists() else None
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)